CHANGES.md

```diff
@@ -1,3 +1,62 @@
+# Version 0.5
+
+## New Features
+
+* Added experimental support for basic, non-compositional verification
+  of x86 machine code for use in conjunction with LLVM verification.
+
+        crucible_llvm_verify_x86 :
+          LLVMModule -> String -> String ->
+          [(String, Int)] -> Bool -> CrucibleSetup () ->
+          TopLevel CrucibleMethodSpec
+
+
+  The first argument specifies the LLVM module containing the _caller_.
+  The second and third specify the ELF file name and symbol name of the
+  function to be verifier. The fourth specifies the names and sizes (in
+  bytes) of global variables to initialize, and the fifth whether to
+  perform path satisfiability checking. The last argument is the LLVM
+  specification of the calling context against which to verify the
+  function.
+
+* Added support for using the SMT theory of arrays in the LLVM memory
+  model. In some cases, this can significantly improve performance.
+
+        enable_smt_array_memory_model : TopLevel ()
+        disable_smt_array_memory_model : TopLevel ()
+
+* Added experimental Verilog export, targeting the subset of Verilog
+  supported by ABC, in the new `write_verilog` and `offline_verilog`
+  commands.
+
+* Added several new options:
+
+    * New functions `enable_what4_hash_consing` and
+      `enable_what4_hash_consing` to enable or disable hash consing to
+      increase sub-formula sharing during symbolic execution.
+
+    * New command-line option `--no-color` to print an ASCII logo
+      without ANSI color or Unicode.
+
+## Performance Improvements
+
+* Improved performance of the LLVM memory model.
+
+* Improved performance of bitvector operations during symbolic execution.
+
+* Improve performance of rewriting SAWCore terms.
```

Comment:
Verb tense should match the other two bullets.

Reply:
Good catch.